src/commands/xsetid.json

```
@@ -54,13 +54,15 @@
                 "name": "entries-added",
                 "token": "ENTRIESADDED",
                 "type": "integer",
-                "optional": true
+                "optional": true,
+                "since": "7.0.0"
             },
             {
                 "name": "max-deleted-id",
```

Let's also modify it to be more consistent
```suggestion
                "name": "max-deleted-entry-id",
```

---

I made the change in the new commit, however let me point out that "entry" was removed in an earlier commit of the file.

---

thanks for spotting this.
@guybe7 it was done in #11051, do you remember the considerations?

---

i removed "entry" so that the name would match the token

i don't really care TBH

---

so let's drop this change and remain consistent with `MAXDELETEDID`.